walk-in-interview/src/main/webapp/account/log-in/business/script.js

```diff
@@ -86,6 +87,8 @@ submitButton.addEventListener('click', (_) => {
         // Enables the button regardless of success or failure
         document.getElementById('submit').disabled = false;
 
+        setCookie(USER_TYPE_COOKIE_PARAM, TYPE_BUSINESS);
```

Comment:
it feels that the server should be responsible for setting the use type cookie as a response to createSession